third_party/gpus/cuda/BUILD.tpl

```diff
@@ -49,8 +49,10 @@ cc_library(
     name = "cudart_static",
     srcs = ["lib/%{cudart_static_lib}"],
     includes = ["include/"],
-    linkopts = [
-        "-ldl",
+    linkopts = select({
+            "//tensorflow:FreeBSD": [],
```

Comment:
This won't find //tensorflow:FreeBSD...

Either the TF team decide to switch to bazel 0.4.4 and you can just add @org_tensorflow in front of the label or you have to work around either by injecting the workspace name from the template generation.